package.json

```diff
@@ -84,6 +87,9 @@
     "json-loader": "0.5.4",
     "node-sass": "3.13.1",
     "precommit-hook": "3.0.0",
+    "react-addons-test-utils": "^15.4.2",
+    "regenerator-runtime": "^0.10.3",
+    "sinon": "^2.1.0",
```

Comment:
Yes, let's define which libraries we're going to use for unit testing and stick to exact version.